sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamExecution.scala

```diff
@@ -160,7 +185,7 @@ class StreamExecution(
         sink.addBatch(nextBatch)
       }
 
-      batchRun = true
+      active = true
```

Comment:
I think `batchRun` is not used anywhere, since the earlier batch marker stuff was removed.